disunity-core/src/test/java/info/ata4/util/collection/NodeTest.java

```diff
@@ -0,0 +1,45 @@
+package info.ata4.util.collection;
+
+import com.diffblue.deeptestutils.Reflector;
+import info.ata4.util.collection.Node;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class NodeTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: Node */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 136 branch to line 136
+   *  - conditional line 137 branch to line 140
+   */
+
+  @Test
+  public void hashCodeOutputPositive() throws InvocationTargetException {
+
+    // Arrange
+    final Node objectUnderTest = (Node)Reflector.getInstance("info.ata4.util.collection.Node");
+    Reflector.setField(objectUnderTest, "children", null);
+    Reflector.setField(objectUnderTest, "data", null);
+    Reflector.setField(objectUnderTest, "parent", null);
+
+    // Act
+    final int retval = objectUnderTest.hashCode();
+
+    // Assert result
+    Assert.assertEquals(9245, retval);
```

Comment:
I've often wondered is it worth including tests that use `hashCode`, as it seems a fairly trivial operation across any object?